tests/wpt/meta/html/semantics/tabular-data/the-table-element/remove-row.html.ini

```diff
@@ -1,3 +0,0 @@
-[remove-row.html]
```

Comment:
You'll need to also run this in legacy layout mode to update the expectation:

```
./mach run-wpt -r /html/semantics/tabular-data/the-table-element/remove-row.html --legacy-layout --log-raw out.log
./mach update-wpt ---legacy-layout out.log
```

I also recommend running the entire directory as I suspect that would have caught the issue above as well.

Reply:
Thank you; updated - apologies I erroneously thought they were both covered by a single file.

> I also recommend running the entire directory as I suspect that would have caught the issue above as well.

This makes a lot of sense 👍 - I regularly see unexpected timeouts when running all tests in a directory, but I can see now it tends to improve after 2-3 runs.

Reply:
Are you are MacOS by chance? There's an issue where the first time you run Servo on MacOS it needs to build the GStreamer plugin database. If you start servo with `./mach run -r` before running tests you shouldn't see so many timeouts. We're still trying to figure out how to avoid this lag on startup.

Reply:
> Are you are MacOS by chance?

I am 👍 I just did this and it works a treat, thank you.